cmd/tke-installer/app/installer/chart.go

```diff
@@ -107,15 +108,14 @@ func (t *TKE) importCharts(ctx context.Context) error {
 			existed, err := client.Push(&helmaction.PushOptions{
 				ChartPathOptions: chartPathBasicOptions,
 				ChartFile:        f,
-				ForceUpload:      false,
+				ForceUpload:      true,
 			})
 			if err != nil {
-				if existed {
-					log.Warn(err.Error())
-				} else {
-					errs = append(errs, err)
-					continue
-				}
+				errs = append(errs, err)
+				continue
+			}
+			if existed {
+				log.Warnf("%s already exist in registry, push with force flag", path.Base(f))
 			}
 		}
 	}
```

Comment:
1. we don't need log message, just force upload chart to chart repo,  this is the default behavior
2. please add comments to the codes for this clarify